### ISSUE TITLE
Issue: [WARNING] Module decompile.py failed to mount!

### ISSUE THREAD

**Issue by davbe** [Jon-Becker/heimdall#6]
**Describe the bug**
Decompiler module cannot be loaded.

**Heimdall Command**
```
python -m heimdall -h
```

***Additional Steps***
None

**Expected behavior**
Module should mount

**Screenshots**
NA

**Desktop (please complete the following information):**
Windows 10 

**Smartphone (please complete the following information):**
NA

**Additional context**
Just git clone the repo and ran the command.

**Comment by davbe** [comment]
My bad, I was missing some requirements...

**Comment by Jon-Becker** [comment]
Make sure you're running the __main__ that's not in the heimdall folder.

PIP messed some stuff up. If you want it to be easy to use and a global command like 

``heimdall -m ...``

use

``pip install eth-heimdall``

**Comment by lior-abadi** [comment]
I am having the same issue with the new release (`1.0.3 stable`).

Tried to run this from the global `pip install` and also from the folder (by executing the `main `that is not in the heimdall folder).

Also I've installed the requirements within `requirements.txt` and still it says that the module decompile failed to mount.

Even when I go to the `-h` section, just `config `and `debug ` are shown but no clue about `decompile`.
